express-post-json/server.js

```diff
@@ -0,0 +1,27 @@
+import express from 'express';
+
+const app = express();
+let nextId = 1;
+const grades = {};
+
+app.get('/api/grades', (req, res) => {
+  const allGrades = [];
+  for (const property in grades) {
+    allGrades.push(grades[property]);
+  }
+  res.json(allGrades);
+});
+
+app.use(express.json());
+
+app.post('/api/grades', (req, res) => {
+  const eachGrade = req.body;
+  eachGrade.id = nextId;
+  grades[nextId] = eachGrade;
+  nextId++;
+  res.status(201).json(grades);
```

Comment:
Oops, the spec says to return the grade that was added, not all the grades. Please fix.